components/Actuators/AutonomousControl/turretTurn.py

```diff
@@ -19,7 +20,7 @@ def setup(self):
     def setAngle(self, angle):
         """sets angle turret is turning to"""
         if self.turretThreshold.angleCheck(angle) != angle:
-            log.error("Turret angle check failed")
+            log.error("Turret angle check failed: "+str(self.turretThreshold.angleCheck(angle))+" vs. "+str(angle))
```

Comment:
```suggestion
            log.error("Turret angle check failed {0} vs. {1}"+self.turretThreshold.angleCheck(angle), angle))
```
To simplify formatting (UNTESTED)